index.html

```diff
@@ -1262,25 +1243,39 @@ <h3>Algorithm</h3>
                 </li>
-                <li class="changed">Otherwise, dereference <var>context</var> using
+                <li class="changed">Otherwise, set <var>context document</var>
+                  to the {{RemoteDocument}} obtained
+                  by dereferencing <var>context</var> using
                   the <a>LoadDocumentCallback</a>, passing <var>context</var>
                   for <a data-link-for="LoadDocumentCallback">url</a>,
                   and <code>http://www.w3.org/ns/json-ld#context</code> for <a data-link-for="LoadDocumentOptions">profile</a>
                   and for <a data-link-for="LoadDocumentOptions">requestProfile</a>.
                   <ol>
                     <li id="alg-context-deref-error">If <var>context</var> cannot be dereferenced,
-                      <span class="changed">or cannot be transformed into the <a>internal representation</a></span>,
+                      <span class="changed">
+                        or the {{RemoteDocument/document}} from <var>context document</var>
+                        cannot be transformed into the <a>internal representation</a>
+                      </span>,
                       a <a data-link-for="JsonLdErrorCode">loading remote context failed</a>
-                      error has been detected and processing is aborted. If the dereferenced document has no
+                      error has been detected and processing is aborted.</li>
+                    <li>If the {{RemoteDocument/document}} has no
                       top-level <a class="changed">map</a> with an <code>@context</code> <a>entry</a>, an
                       <a data-link-for="JsonLdErrorCode">invalid remote context</a>
                       has been detected and processing is aborted.</li>
-                    <li>Otherwise, set <var>context</var> to the value of that <a>entry</a>.</li>
+                    <li>Set <var>loaded context</var> to the value of that <a>entry</a>.</li>
                   </ol>
                 </li>
                 <li>Set <var>result</var> to the result of recursively calling this algorithm,
                   passing <var>result</var> for <var>active context</var>,
-                  <var>context</var> for <var>local context</var>, and <span class="changed">a copy of</span> <var>remote contexts</var>.</li>
+                  <var>loaded context</var> for <var>local context</var>,
+                  <span class="changed">
+                    the {{RemoteDocument/documentUrl}} of <var>context document</var> for <var>base URL</var>,
+                  </span>
+                  and <span class="changed">a copy of</span> <var>remote contexts</var>.
+                  <div class="note">If <var>context</var> was previously dereferenced,
+                    <a>processors</a> MUST make provisions for retaining the <var>base URL</var>
+                    of that <a>context</a> for this step.</div>
```

Comment:
Can we say why? (I presume to enable the resolution of any relative context URLs that may be encountered during processing).

```suggestion
                    of that <a>context</a> for this step to enable the resolution of any
                    relative context URLs that may be encountered during processing.</div>
```